app/models/connected_stripe_account.rb

```diff
@@ -70,14 +70,16 @@ def prepare_intent(registration, amount_iso, currency_iso, paying_user)
     )
   end
 
-  def issue_refund(charge_id, amount_iso)
-    charge_record = StripeRecord.charge.find_by!(stripe_id: charge_id)
+  def find_payment_record(record_id)
```

Comment:
Is the naming here appropriate given that we're only returning `charge` payment record objects? I'd expect a method like this to either not limit its search to a scope, or take a `record_type` parameter which defines which scope will be searched.

Reply:
Renamed.